### PR TITLE
made models

### PR DESCRIPTION
**Pull request by reubenfrith**
# Created all models

## Description

Created all the models refer to schema https://kitt.lewagon.com/db/86289
### Resources used

chat gpt hehe

## Types of change

- [x] New feature (change which adds functionality)

- [ ] Bug fix (change which fixes an issue)

- [ ] Improvement/refactoring (change that doesn't add any features but makes things better)

- [ ] Styling (style changes that don't affect functionality)

## Checklist before submitting

- [ ] Satisfies card on Trello

- [ ] Has had a another team mate review your design.

- [x] No known defects

- [ ] Appropriate tests completed (e.g. core functionality, edge cases)

- [ ] Code has been formatted with nicely and contains proper indentation.

- [ ] Code comments as appropriate

- [ ] `master` branch has been merged into the feature branch and tested (if any newer commits on `master`)

### CODE REVIEW

**Review by Zhenni-Z — APPROVED** [review]
Looks great!

File: db/migrate/20230103090512_create_links.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateLinks < ActiveRecord::Migration[7.0]
```
Comment (drenetic) [inline db/migrate/20230103090512_create_links.rb:1]: We'll amend this later to include a 'time' field

**Review by drenetic — APPROVED** [review]
Looks good!

**Review by drenetic — CHANGES_REQUESTED** [review]
Requires 'has_many' relationships

**Review by drenetic — APPROVED** [review]

**Review by drenetic — APPROVED** [review]
